app/user_test.go

```diff
@@ -1394,3 +1394,18 @@ func TestUpdateUserRolesWithUser(t *testing.T) {
 	_, err = th.App.UpdateUserRolesWithUser(user, "does not exist", false)
 	require.NotNil(t, err)
 }
+
+func TestDeactivateMfa(t *testing.T) {
+	t.Run("MFA is disabled", func(t *testing.T) {
+		th := Setup(t).InitBasic()
+		defer th.TearDown()
+
```

Comment:
Is it possible to enable MFA here and assign the user some fake MFA code and then turn off MFA and see if the user can deactivate?
This will more closely mimic the scenario explained in the ticket. Not sure if it's possible but would recommend to look into if it is.

Reply:
The `App.ActivateMfa` method calls the MFA module directly, which then calls `dgoogauth.OTPConfig.Activate`. We would probably have to create mock interfaces for the MFA module if we want to enable MFA in our test.